src/EA.Iws.Domain.Tests.Unit/Movement/CapturedMovementFactoryTests.cs

```diff
@@ -7,20 +7,27 @@
     using Domain.Movement;
     using Domain.NotificationAssessment;
     using FakeItEasy;
+    using Prsd.Core;
     using Prsd.Core.Domain;
     using TestHelpers.DomainFakes;
-    using Xunit;
+    using Xunit;   
 
     public class CapturedMovementFactoryTests
     {
         private static readonly Guid NotificationId = new Guid("675287E3-42D3-4A58-86D4-691ECF620671");
         private static readonly DateTime AnyDate = new DateTime(2015, 1, 1);
+        private static readonly DateTime Today  = new DateTime(2017, 07, 19);
+        private static readonly DateTime PastDate = Today.AddDays(-2);
+        private static readonly DateTime FutureDate = Today.AddDays(2);
+        private static readonly DateTime FutureSixtyDate = Today.AddDays(60);
+        private static readonly DateTime FutureSixtyOneDate = Today.AddDays(61);
         private readonly ICapturedMovementFactory factory;
         private readonly IMovementNumberValidator validator;
         private readonly INotificationAssessmentRepository assessmentRepository;
 
         public CapturedMovementFactoryTests()
         {
+            SystemTime.Freeze(Today);
```

Comment:
You need to implement IDisposable and call SystemTime.Unfreeze() in the Dispose() method